### ISSUE TITLE
Adjust example in `get_accessibility()` function

### ISSUE THREAD

**Issue by rafavdz** [urbanbigdatacentre/AccessUK#2]
Currently, the example in the documentation of the  `get_accessibility()` function does not work as the 'from' argument is NULL by default and it requires a reference to run. It should be:

  ```
## Not run: 
glasgow_cc = c("S01010272", "S01010265")
get_accessibility(from = glasgow_cc)

## End(Not run)
```
